Makefile

```diff
@@ -1,15 +1,16 @@
-GO_BUILD=GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags="-s -w"
-RELEASE_VERSION = "2.0.0"
-DOCKER_HUB_USER = "iguaziodocker"
+LOCATOR_TAG = 2.0.0
 
-all: lint bin image
+.PHONY: all
+all: lint build
 	@echo Done.
 
-bin: ensure-gopath
-	$(GO_BUILD) -o locatorctl cmd/locatorctl/main.go
+.PHONY: build
+build:
+	docker build --tag=$(LOCATOR_REPOSITORY)locator:$(LOCATOR_TAG) .
 
-image:
-	docker build --rm --tag $(DOCKER_HUB_USER)/locator:$(RELEASE_VERSION) .
+.PHONY: ensure-gopath bin
+bin:
+	CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags="-s -w" -o $(GOPATH)/bin/locatorctl $(GOPATH)/src/github.com/v3io/locator/cmd/locatorctl/main.go
```

Comment:
I know I'm picky here. But just use the original target with the var. I prefer the shorter line.
Same goes of you can extract project path to var.

Reply:
Fixed